presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -845,3 +845,40 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+CTE Materialization Properties
+--------------------------------------
+
+``cte-materialization-strategy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Allowed values:** ``ALL``, ``NONE``
+    * **Default value:** ``NONE``
+
+    Specifies the strategy to use for materializing Common Table Expressions (CTEs) in queries.
+    ``NONE`` indicates that no CTEs will be materialized.
+    ``ALL`` indicates that all CTEs in the query will be materialized.
+    This can also be specified on a per-query basis using the ``cte_materialization_strategy`` session property.
+
+``query.cte-hash-partition-count``
+^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``100``
+
+    The number of partitions to be used for materializing Common Table Expressions (CTEs) in queries.
+    This setting determines how many buckets or writers should be used when materializing the CTEs, potentially affecting the performance of queries involving CTE materialization.
+    A higher number of partitions might improve parallelism but also increases overhead in terms of memory and network communication.
+    Recommended value: 4 - 10x times the size of the cluster
```

Comment:
```suggestion
    Recommended value: 4 - 10x times the size of the cluster.
```
Without the period the paragraph looks like this in a new build
<img width="1041" alt="Screenshot 2024-01-22 at 4 47 32 PM" src="https://github.com/prestodb/presto/assets/7013443/dd238099-4f44-49c7-8bca-2f824c2798dd">